### PR TITLE
Delete old dev parts

### PR DESCRIPTION
**Pull request by submarcos**
and clean urls and requirements

### CODE REVIEW

File: geostore/urls.py [hunk 1]
```diff
@@ -1,69 +1,45 @@
...
-router = routers.SimpleRouter()
-
-router.register(r'layer', LayerViewSet)
-router.register(r'layer/(?P<layer>[\d\w\-_]+)/feature', FeatureViewSet,
-                base_name='feature')
-router.register(r'layer_relation', LayerRelationViewSet)
-router.register(r'layer_relation/(?P<layerrelation_pk>\d+)/feature_relation',
-                FeatureRelationViewSet)
-
-urlpatterns += router.urls
```
Comment (nahuelange) [inline geostore/urls.py:58]: Why delete this ?

Comment (submarcos) [reply geostore/urls.py:58]: best practice is to use include() instead of +=

Comment (nahuelange) [reply geostore/urls.py:58]: Oh sorry, I didn't see the include!

**Review by nahuelange — APPROVED** [review]

File: geostore/urls.py (hunk 1 above)
Comment (submarcos) [reply geostore/urls.py:58]: path('', include(router.urls)),

**Pull request merged**